addons/mrp_account/i18n/vi.po

```diff
@@ -210,12 +210,12 @@ msgstr "Số đơn công đoạn"
 #. module: mrp_account
 #: model:ir.model,name:mrp_account.model_mrp_workcenter_productivity
 msgid "Workcenter Productivity Log"
-msgstr "Nhật ký sản xuất của Năng lực sản xuất"
+msgstr "Nhật ký Khu vực Sản xuất"
 
 #. module: mrp_account
 #: model:ir.model.fields,field_description:mrp_account.field_account_analytic_account__workcenter_ids
 msgid "Workcenters"
-msgstr "Năng lực Sản xuất"
+msgstr "Khu vực Sản xuất"
```

Comment:
```suggestion
msgstr "Năng lực Sản xuất"
```

Reply:
Thưa anh phần này em thấy từ workcenter để thành khu vực sản xuất sẽ chuẩn hơn vì nó mang nghĩa của một nơi sản xuất vật lý như là 1 dây chuyền , 1 nhà máy 
mong anh xem lại giúp em ạ

Reply:
Các tài liệu nhà mình đang thống nhất sử dụng thuật ngữ "Năng lực Sản xuất" cho workcenter.

1. https://viindoo.com/documentation/15.0/vi/applications/supply-chain/manufacturing/operations/create-and-use-alternative-work-centers.html
2. https://viindoo.com/vi/forum/tro-giup-1/cach-tich-hop-phan-mem-quan-ly-sua-chua-va-bao-hanh-va-phan-mem-quan-ly-san-xuat-4186
3. https://erponline.vn/vi/forum/hoi-dap-7/thiet-lap-thong-so-cho-nang-luc-san-xuat-workcenter-2480
4. v.v.

Từ này cũng chưa chuẩn lắm như có lẽ khi nào có điều kiện ta sẽ xem xét đổi hàng loạt sau.

Reply:
> Các tài liệu nhà mình đang thống nhất sử dụng thuật ngữ "Năng lực Sản xuất" cho workcenter.
> 
> 1. https://viindoo.com/documentation/15.0/vi/applications/supply-chain/manufacturing/operations/create-and-use-alternative-work-centers.html
> 2. https://viindoo.com/vi/forum/tro-giup-1/cach-tich-hop-phan-mem-quan-ly-sua-chua-va-bao-hanh-va-phan-mem-quan-ly-san-xuat-4186
> 3. https://erponline.vn/vi/forum/hoi-dap-7/thiet-lap-thong-so-cho-nang-luc-san-xuat-workcenter-2480
> 4. v.v.
> 
> Từ này cũng chưa chuẩn lắm như có lẽ khi nào có điều kiện ta sẽ xem xét đổi hàng loạt sau.

Dạ vâng ạ , vậy PR mrp kia em đổi lai theo luôn ạ